File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCS1003R00LayOUT1001Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCS1003R00LayOUT1001Info
    {
        private String _reserYn;
        private String _bunho;
        private String _suname;
        private String _suname2;
        private String _birth;
        private String _sexAge;
        private String _doctor;
        private String _doctorName;
        private String _gwa;
        private String _gwaName;
        private String _chojae;
        private String _chojaeName;
        private String _naewonDate;
        private String _inputGubun;
        private String _orderEndYn;

        public String ReserYn
        {
            get { return this._reserYn; }
            set { this._reserYn = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Suname2
        {
            get { return this._suname2; }
            set { this._suname2 = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public String SexAge
        {
            get { return this._sexAge; }
            set { this._sexAge = value; }
        }

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String Chojae
        {
            get { return this._chojae; }
            set { this._chojae = value; }
        }

        public String ChojaeName
        {
            get { return this._chojaeName; }
            set { this._chojaeName = value; }
        }

        public String NaewonDate
        {
            get { return this._naewonDate; }
            set { this._naewonDate = value; }
        }

        public String InputGubun
        {
            get { return this._inputGubun; }
            set { this._inputGubun = value; }
        }

        public String OrderEndYn
        {
            get { return this._orderEndYn; }
            set { this._orderEndYn = value; }
        }

        public OCS1003R00LayOUT1001Info() { }

        public OCS1003R00LayOUT1001Info(String reserYn, String bunho, String suname, String suname2, String birth, String sexAge, String doctor, String doctorName, String gwa, String gwaName, String chojae, String chojaeName, String naewonDate, String inputGubun, String orderEndYn)
        {
            this._reserYn = reserYn;
            this._bunho = bunho;
            this._suname = suname;
            this._suname2 = suname2;
            this._birth = birth;
            this._sexAge = sexAge;
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._chojae = chojae;
            this._chojaeName = chojaeName;
            this._naewonDate = naewonDate;
            this._inputGubun = inputGubun;
            this._orderEndYn = orderEndYn;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS1003R00LayOUT1001Info.java
package nta.med.data.model.ihis.ocso;

import java.util.Date;

public class OCS1003R00LayOUT1001Info {
	private String reserYn                  ;
	private String bunho                    ;
	private String suname                   ;
	private String suname2                  ;
	private String birth                    ;
	private String sexAge                  ;
	private String doctor                   ;
	private String doctoName              ;
	private String gwa                      ;
	private String gwaName                 ;
	private String chojae                   ;
	private String chojaeName              ;
	private String naewonDate              ;
	private String inputGubun              ;
	private String orderEndYn             ;
	public OCS1003R00LayOUT1001Info(String reserYn, String bunho,
			String suname, String suname2, String birth, String sexAge,
			String doctor, String doctoName, String gwa, String gwaName,
			String chojae, String chojaeName, String naewonDate,
			String inputGubun, String orderEndYn) {
		super();
		this.reserYn = reserYn;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
		this.birth = birth;
		this.sexAge = sexAge;
		this.doctor = doctor;
		this.doctoName = doctoName;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.chojae = chojae;
		this.chojaeName = chojaeName;
		this.naewonDate = naewonDate;
		this.inputGubun = inputGubun;
		this.orderEndYn = orderEndYn;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
	public String getSexAge() {
		return sexAge;
	}
	public void setSexAge(String sexAge) {
		this.sexAge = sexAge;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctoName() {
		return doctoName;
	}
	public void setDoctoName(String doctoName) {
		this.doctoName = doctoName;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getChojae() {
		return chojae;
	}
	public void setChojae(String chojae) {
		this.chojae = chojae;
	}
	public String getChojaeName() {
		return chojaeName;
	}
	public void setChojaeName(String chojaeName) {
		this.chojaeName = chojaeName;
	}
	public String getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(String naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getInputGubun() {
		return inputGubun;
	}
	public void setInputGubun(String inputGubun) {
		this.inputGubun = inputGubun;
	}
	public String getOrderEndYn() {
		return orderEndYn;
	}
	public void setOrderEndYn(String orderEndYn) {
		this.orderEndYn = orderEndYn;
	}
	
}
